examples/acquiring_data/querying_the_GOES_event_list.py

```diff
@@ -51,6 +51,18 @@
 new_table = hek_results["event_starttime", "event_peaktime",
                         "event_endtime", "fl_goescls", "ar_noaanum"]
 
+###################################################################
+# ``hek_result`` is already sorted by date, if one wants to sort
+# by magnitude, one can do the following:
+
+by_magnitude = sorted(hek_results, key=lambda x: ord(x['fl_goescls'][0])*100+float(x['fl_goescls'][1:]))
```

Comment:
```suggestion
# Sorting is done by using the flare class from "fl_goescls"
# By converting the flare class to a number using ord()
# and adding the flare strength, we can sort by value
by_magnitude = sorted(hek_results, key=lambda x: ord(x['fl_goescls'][0]) + float(x['fl_goescls'][1:]))
```
Do we need the *100?
This is also sorting in low to high, I think we want high to low.